parm/config/config.fcst

```diff
@@ -255,14 +325,17 @@ elif [[ "$CDUMP" == "gfs" ]] ; then # GFS cycle specific parameters
       export io_layout="1,1"
     fi
 
-
 fi
 
-    ## Debug load balancing
-    # export KEEPDATA="YES"
-    # export ESMF_RUNTIME_PROFILE=ON
-    # export ESMF_RUNTIME_PROFILE_OUTPUT=SUMMARY
-
+if [ $cplgocart = .true. ]; then # temporary settings for aerosol coupling
```

Comment:
Shouldn't this use DO_GOCART instead, per previous conversations?

Reply:
Yes, have now changed it to DO_GOCART=YES. Thanks!